go.mod

```diff
@@ -7,25 +7,25 @@ require (
 	github.com/btcsuite/btcd/btcutil/psbt v1.1.5
 	github.com/btcsuite/btcd/chaincfg/chainhash v1.0.2
 	github.com/btcsuite/btclog v0.0.0-20170628155309-84c8d2346e9f
-	github.com/btcsuite/btcwallet v0.16.6-0.20221208210930-6f3f55efb230
+	github.com/btcsuite/btcwallet v0.16.7
 	github.com/btcsuite/btcwallet/wtxmgr v1.5.0
 	github.com/coreos/bbolt v1.3.3
 	github.com/davecgh/go-spew v1.1.1
 	github.com/decred/dcrd/dcrec/secp256k1/v4 v4.0.1
 	github.com/fortytw2/leaktest v1.3.0
 	github.com/grpc-ecosystem/grpc-gateway/v2 v2.5.0
 	github.com/jessevdk/go-flags v1.4.0
-	github.com/lightninglabs/aperture v0.1.18-beta
+	github.com/lightninglabs/aperture v0.1.19-beta.0.20230227124349-b952e4ee6ff4
```

Comment:
Maybe we can also tag `aperture` so we have a proper tag here.

Reply:
My plan was to tag aperture as soon as we bumped it to `lnd v0.16.0-beta` final. Otherwise if we have the RC in all the `go.mod` files, go mod will want to use that instead of the final one in litd. So as soon as the final release is out, we'll bump to that version in all projects and then tag for inclusion in litd.